### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by nuria-gs**
### ISSUE NUMBER
#49 Create an initial header 

### Labels
- IN REVIEW

### Describe the changes
Added the HTML for the header and some basic styling.

### Screenshots (if appropriate)
header mobile
header large

### Questions or comments (if any)
n/a

### CODE REVIEW

File: frontend/sass/partials/_header.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        margin-right: 2em;
+     }
+  }
+  
+  &__link{
+	  list-style-type: none;
+    display: inline;
+    margin: 1em;
+    
+    a{
```
Comment (lili2311) [inline frontend/sass/partials/_header.scss:40]: weird spacing here, please amend

File: frontend/sass/partials/_header.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    margin: 0;
+    
+     @media(min-width:768px){
+        padding: 2em;
+        margin-right: 2em;
+     }
+  }
+  
+  &__link{
+	  list-style-type: none;
```
Comment (lili2311) [inline frontend/sass/partials/_header.scss]: also too much space here

**Review by lili2311 — APPROVED** [review]
Just some spacing is odd in a couple of places otherwise good to merge

File: frontend/sass/partials/_header.scss (hunk 2 above)
Comment (nuria-gs) [reply frontend/sass/partials/_header.scss]: Indentation corrected :)

File: frontend/sass/partials/_header.scss (hunk 1 above)
Comment (nuria-gs) [reply frontend/sass/partials/_header.scss:40]: Indentation corrected :)

**Comment by lili2311** [comment]
👍  but can you please solve the merge conflicts before merging it wont let you merge until then.

**Comment by nuria-gs** [comment]
I haven't actually modify and committed changes on these files. Not sure if it's because of the "grunt default".
Not sure if I have to merge the files, or just discard my changes and run the "grunt default" in the master branch

**Comment by lili2311** [comment]
@nuria-gs you can follow the instructions it gives you `Use the command line` click on that and try it out.

screen shot 2016-12-07 at 22 34 47

The files tab has both of these files commited with changes, this is completely normal it just means that github cannot automatically detect how to merge the changes all together. You will need to copy the new version of `styles.css` before you begin this and paste it over styles.css to say you want to keep all your changes as they are generated. We can remove this file form git completely as it is generated

**Comment by nuria-gs** [comment]
Ok, done!

**Pull request merged**